### ISSUE TITLE
Old segment files are not removed when service is stopped and started again

### ISSUE THREAD

**Issue by johnwcassidy** [jaredpetersen/raspi-live#39]
Steps to reproduce:
1. Start service
2. Stream for a few moments so some file segments are added to wherever they are configured to go
3. stop service
4. start service
5. stream for a few moments again

Expected result:
orphaned segments are cleaned up

Actual result:
orphaned segments remain

This is a concern for the configuration I currently have, where i setup a system service to execute `raspi-live`, and control when it starts and stops with `cron`. With each stop and start, I'm accumulating orphaned segment files that I will need to clean up manually.

<img width="396" alt="Screen Shot 2020-05-14 at 11 39 24 PM" src="https://user-images.githubusercontent.com/48183097/82009043-42bb5980-963c-11ea-8ed3-a8372c287389.png">

**Comment by jaredpetersen** [comment]
Yeah, we'd have to look at refactoring how these get cleaned up. Right now, FFMpeg handles that completely.

An easy fix would just be that raspi-live removes everything in the directory on startup so that it starts with a clean slate.
